apps/backend/src/datasources/postgres/FapDataSource.ts

```diff
@@ -894,17 +894,27 @@ export default class PostgresFapDataSource implements FapDataSource {
   async removeMemberFromFapProposal(
     proposalPk: number,
     fapId: number,
-    memberId: number
+    memberId: number,
+    reviewId: number
   ) {
-    const memberRemovedFromProposal = await database('fap_assignments')
-      .del()
-      .where('fap_id', fapId)
-      .andWhere('proposal_pk', proposalPk)
-      .andWhere('fap_member_user_id', memberId);
+    await database.transaction(async (trx) => {
+      await trx
+        .from('fap_reviews')
+        .where('review_id', reviewId)
+        .returning('*')
+        .del();
```

Comment:
Rather than add a reviewId parameter, could we use the existing filtering arguments? This table already has equivalent `fap_id`, `proposal_pk` and `user_id` columns to match the ones filtered on in the fap_assignments delete, and that should ensure that an API client can't call this mutation with a `reviewId` that doesn't line up with the other parameters.